server/settings/main.py

```diff
@@ -51,8 +51,8 @@ class MayaSettings(BaseSettingsModel):
     )
     ext_mapping: list[ExtMappingItemModel] = SettingsField(
         default_factory=list, title="Extension Mapping")
-    maya_dirmap: MayaDirmapModel = SettingsField(
-        default_factory=MayaDirmapModel, title="Maya dirmap Settings")
+    dirmap: DirmapModel = SettingsField(
```

Comment:
Note, if anyone did use it, we probably should add settings overrides conversion (don't know if it did work tho?).

Reply:
It didn't work - because it was using the wrong key in `ayon-core` all this time. Last time it would've worked was OpenPype then.

Reply:
But it was used in `prepare_root_value` too, so people might have filled the values, or?

Reply:
Should be implemented with https://github.com/ynput/ayon-maya/pull/202/commits/5b12acbcf12283afdfbbae505126614b8cb4e77d